src/sequence_batch.h

```diff
@@ -81,6 +81,25 @@ class SequenceBatch {
     return negative_sequence_batch_[sequence_index];
   }
 
+  // big_endian: N_pos is in the order of sequence
+  // little_endian: N_pos is in the order from the sequence right side to left, 
+  //                this is the order of the GenerateSeed
+  inline void GetSequenceNsAt(uint32_t sequence_index, bool little_endian, std::vector<int> &N_pos) {
+    int l = sequence_batch_[sequence_index]->seq.l;
```

Comment:
Make this const if it won't change, which help avoid potential bugs. We will do this for future code and when I refactor the current code.

Reply:
Done.